### ISSUE TITLE
Testruns using docker container for the latest (1.10.0-rc.1) candidate give babel runtime errors

### ISSUE THREAD

**Issue by raj-balaebail** [DevExpress/testcafe#5709]
### What is your Test Scenario?

 Run tests in docker container
### What is the Current behavior?

 ```
$ docker-compose -f docker-compose.yml -p buildkite806bfe3f2e8b4a85ace8c7b98b5436dc run --name buildkite806bfe3f2e8b4a85ace8c7b98b5436dc_testcafe_build_11 --rm testcafe
--
  | The "src" and "browsers" options from the configuration file will be ignored.
  | ERROR Cannot prepare tests due to an error.
  |  
  | Error: [BABEL] /tests/tests/td_tests/rates_terms_page/rates-terms-page-FL-tests.spec.js: Failed to resolve "@babel/runtime" relative to "/tests" (While processing: "base$0")

```
### What is the Expected behavior?

 Runs the tests 
### What is your web application and your TestCafe test code?

 
<details>
<summary>Your website URL (or attach your complete example):</summary>

</details>

<details>
<summary>Your complete test code (or attach your test files):</summary>

 
```js
 
```
</details>

<details>
<summary>Your complete configuration file (if any):</summary>

```

```
</details>

<details>
<summary>Your complete test report:</summary>

```

```
</details>

<details>
<summary>Screenshots:</summary>

```

```
</details>

### Steps to Reproduce:

1. Go to my website ...
3. Execute this command...
4. See the error...
 
### Your Environment details:
 
* testcafe version:                   
* node.js version:                    
* command-line arguments:  
* browser name and version: 
* platform and version:          
* other:

**Comment by miherlosev** [comment]
Hi @raj-balaebail 

Thank you for the shared information. I've reproduced the problem.

**Comment by AndreyBelym** [comment]
Fixed by #5753
